packages/editor/src/components/provider/index.js

```diff
@@ -195,12 +187,14 @@ export const ExperimentalEditorProvider = withRegistryProvider(
 					}
 				);
 			}
+			// Not sure if the missing no dependencies here is deliberate or not so ignoring lint warning for now.
+			// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Let's keep the warning around so (remove the disable comment) that way we investigate it in the future.

Reply:
Done.